packages/admin/cms-admin/src/pages/createEditPageNode.tsx

```diff
@@ -313,7 +314,19 @@ export function createEditPageNode({
                                         }
 
                                         if (values.slug === "home") {
-                                            return <Typography>/</Typography>;
+                                            return (
+                                                <>
+                                                    <FieldContainer
+                                                        label={intl.formatMessage({
+                                                            id: "comet.pages.pages.page.path",
+                                                            defaultMessage: "Complete Path",
+                                                        })}
+                                                        variant="horizontal"
+                                                    >
+                                                        <Typography>/</Typography>
+                                                    </FieldContainer>
+                                                </>
```

Comment:
Do you need that `if`? Can't you do that down below in the return?

Somewhat like 

```tsx
<FieldContainer
    label={intl.formatMessage({
        id: "comet.pages.pages.page.path",
        defaultMessage: "Complete Path",
    })}
    variant="horizontal"
>
    <Typography>
        {values.slug === "home" ? "/" : `${parentPath}/${values.slug}`}
    </Typography>
</FieldContainer>
```

Reply:
Also the fragment is not necessary because you're only returning one element

Reply:
I wasn't sure whether the code after the FieldContainer should also run when it's _home_, so I left the _if_ for the time being just to be safe. Will do 👍